docs/directory_structures.md

```diff
@@ -0,0 +1,76 @@
+# Directory structures for Lightning Pose
+
+## Data directory structure
+
+Lightning Pose requires data to be stored in a particular manner.
+Also see the example dataset provided in `lightning-pose/data/mirror-mouse-example`.
+
+    /path/to/project/
+      ├── labeled-data/
+      ├── videos/
+      └── CollectedData.csv
+
+* `CollectedData.csv`: hand labels in human-readable format (more below). 
+Note that this file can take any name, and needs to be specified in the config file under 
+`data.csv_file`.
+
+* `labeled-data/`: contains images that correspond to the labels, and can include subdirectories.
+The directory name, any subdirectory names, and image names are all flexible, as long as they are
+consistent with the first column of `CollectedData.csv`.
+
+* `videos/`: when training semi-supervised models, the videos in this directory will be used for 
+computing the unsupervised losses. This directory can take any name, and needs to be specified in
+the config file under `data.video_dir`.
+
+## Converting DLC projects to Lightning Pose format
+Once you have installed Lightning Pose, you can convert previous DLC projects into the proper 
+Lightning Pose format by running the following script from the command line 
+(make sure to activate the conda environment):
+```console
+python scripts/converters/dlc2lp.py --dlc_dir=/path/to/dlc_dir --lp_dir=/path/to/lp_dir
+```
+That's it! After this you will need to update your config file with the correct paths.
+
+### Converting other projects to Lightning Pose format
+Coming soon. If you have labeled data from other pose estimation packages (like SLEAP or DPK) and
+would like to try out Lightning Pose, please 
+[raise an issue](https://github.com/danbider/lightning-pose/issues).
+
+## Model directory structure
+
+If you train a model using our script `lightning-pose/scripts/train_hydra.py`, a directory will be
```

Comment:
check the deprecation warnings we get with this -- i'm not sure that this convention will stay for long

Reply:
we specify this explicitly in `config_default.yaml` so hopefully even after hydra updates this will be true?